src/CDN.js

```diff
@@ -15,7 +15,7 @@ class CDN {
       'font-awesome': '4.7.0',
       'grid': 'latest',
       'moment-timezone': 'latest',
-      'quill': '2.0.0-dev.3',
+      'quill': '2.0.0-dev.4',
```

Comment:
When I open https://cdn.test-form.io/quill/2.0.0-dev.4/quill.js, I still see 2.0.0-dev.3. Are we going to upgrade it?

Reply:
As I understand it, we just need to update our https://cdn.test-form.io/quill/2.0.0-dev.3/quill.min.js because now 1.3.7 version is stored there. Also I think we need to update our https://cdn.test-form.io/tree.json